resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,152 @@
+"""Converting bam to bed and normalised bigwig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    Process,
+    SchedulingClass,
+    FileField,
+    StringField
+)
+
+
+class BamToBed(Process):
+    """Takes in a bam file and calculates a normalization factor in .bed format.
+    Done by sorting with samtools and transformed with bedtools.
+    """
+
+    slug = "bam-to-bed"
+    name = "Bam to bed"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Badtools bamtobed ({{alignment|sample_name}})"
```

Comment:
I think it's `bedtools`, not `badtools`.

You can add some more defaults to the name: `alignment|sample_name|default('?')`